docker-compose.yml

```diff
@@ -13,8 +13,9 @@ services:
     environment:
       MYSQL_DATABASE: ${DB_NAME}
       MYSQL_ROOT_PASSWORD: ${DB_PASS}
-      MYSQL_USER: ${DB_USER}
-      MYSQL_PASSWORD: ${DB_PASS}
+      MYSQL_ALLOW_EMPTY_PASSWORD: 1
+    env_file:
+      - .env
```

Comment:
The `.env` file is loaded by default I think

Reply:
That would be great, I'm unable to find documentation about this though :/

Reply:
Seems like it does though, but maybe it's still nice to be explicit?

Reply:
Sure :+1: